templates/pipelines/common-app-ci.yaml

```diff
@@ -289,6 +291,11 @@ stages:
           value: ${{ parameters.serviceName }}
         - name: PSHelperDirectory
           value: '$(Pipeline.Workspace)/s/PipelineCommon/templates/powershell/modules/ps-helpers'
+        - name: configFileType
+          ${{ if and(ne(parameters.appDeployConfig, ''),eq(parameters.appDeployConfig.filetype, '')) }}:          
```

Comment:
This if condition can be changed to use yaml coalesce function

Reply:
Updated the code

Reply:
Updated